### ISSUE TITLE
Tailing does not work anymore

### ISSUE THREAD

**Issue by derrapf** [notepad-plus-plus/notepad-plus-plus#14684]
### Description of the Issue
The option view/(tail -f)  did work great until one of the lates updates was done
No I only see changes in the status line, where it tells me the lemgth of the file, but the view port stays as it is.
No tailing of the current file

### Steps to Reproduce the Issue
1.  Load a file 
2.  Set option tail -f
3. change it externally 
4. ->No tailing
5. You even can not klick any menu. Notepad++ is frozen until there is no external change of the file anymore
6. 

### Expected Behavior
View port should be constantly updated
### Actual Behavior
No update of the file

### Debug Information
Notepad++ v8.6.2   (32-bit)
Build time : Jan 14 2024 - 02:18:41
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : "C:\Users\rz\AppData\Local\Temp\DPLPUpdate.log"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 11 Enterprise (64-bit)
OS Version : 22H2
OS Build : 22621.3007
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.1)
    mimeTools (3)
    NppConverter (4.5)
    NppExport (0.4)
    RunMe (1.6)

**Comment by xomx** [comment]
It works ok for me (tested with fresh N++ x86 & x64 v8.6.2 portable, Win10Ent).

Here is my batch file for testing:
[test_for_tail-f.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/14191942/test_for_tail-f.zip)

**Comment by derrapf** [comment]
Yes that is strange. I tryed some other computers and could not reproduce it. Maybe a graphic issue on the VM where I tried...
I will observe that problem and try to find out what could be the difference between those boxes. I hope I can reproduce it.
